projects/components/src/table/cells/data-renderers/icon/icon-table-cell-renderer.component.ts

```diff
@@ -21,4 +23,17 @@ import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
   alignment: TableCellAlignmentType.Center,
   parser: CoreTableCellParserType.Icon
 })
-export class IconTableCellRendererComponent extends TableCellRendererBase<string> {}
+export class IconTableCellRendererComponent extends TableCellRendererBase<CellData, Value> implements OnInit {
+  public iconSize: IconSize = IconSize.Small;
+
+  public ngOnInit(): void {
+    super.ngOnInit();
+
+    if (this.value && this.value.size) {
+      this.iconSize = this.value?.size;
+    }
```

Comment:
nit:
```suggestion
    this.iconSize = this.value?.size ?? IconSize.Small;
```

Reply:
Hah, this is exactly what I had before and changed it last moment.